packages/twenty-front/src/modules/ui/field/display/components/BooleanDisplay.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconCheck, IconX } from 'twenty-ui';
+
+const StyledBooleanFieldValue = styled.div`
+  margin-left: ${({ theme }) => theme.spacing(1)};
+`;
+
+type BooleanDisplayProps = {
+  value: boolean | null;
+};
+
+export const BooleanDisplay = ({ value }: BooleanDisplayProps) => {
+  const theme = useTheme();
+
+  return (
+    <>
+      {value ? (
```

Comment:
@ijreilly I think we prefer to explicitly use isDefined in the FE for this case 👍

Reply:
Actually isDefined = `!isUndefined(value) && !isNull(value)` maybe that's not what you want. You can check `!isUndefined` instead if you don't want to display the icon